### PR TITLE
Custom music clarification

### PR DESCRIPTION
**Pull request by Blizzardsev**
- Added clarification on custom music

### CODE REVIEW

File: game/script-topics.rpy [hunk 1]
```diff
@@ -4406,6 +4406,11 @@ label talk_custom_music_explanation:
     n 1ullss "If you don't know how to check,{w=0.1} then just look at the letters after the period in the file name."
     n 1unmss "You should also be able to see those in the file {i}properties{/i} if they don't appear on the screen at first."
     n 1flrbg "Like I said -{w=0.1} this thing isn't {i}exactly{/i} super modern,{w=0.1} so it won't work with any fancy newer formats,{w=0.1} or weird old ones."
+    n 1uwdaj "Oh,{w=0.75}{nw}"
+    extend 1nlrpu " and if you gotta convert it first,{w=1}{nw}"
```
Comment (Booplicate) [inline game/script-topics.rpy:4410]: Are you sure about using `gotta` here? Maybe `gonna`? (if sure, ignore this)

File: game/script-topics.rpy [hunk 2]
```diff
@@ -4406,6 +4406,11 @@ label talk_custom_music_explanation:
...
+    extend 1nsqpo " don't just rename it."
+    n 1fcsbg "Use a proper converter!"
```
Comment (Booplicate) [inline game/script-topics.rpy:4412]: Next line uses `extend`, but you have no `w` or `nw` tags here, maybe a mistake?

File: game/script-topics.rpy [hunk 3]
```diff
@@ -4406,6 +4406,11 @@ label talk_custom_music_explanation:
...
+    extend 1fsrbg " Unless you {i}like{/i} hearing your music being all warped and nasty,{w=0.3} anyway."
```
Comment (Booplicate) [inline game/script-topics.rpy:4413]: I'd use `enjoy` instead of `like`, but up to you.

File: game/script-topics.rpy (hunk 2 above)
Comment (Blizzardsev) [reply game/script-topics.rpy:4412]: Good catch - thanks!

File: game/script-topics.rpy (hunk 3 above)
Comment (Blizzardsev) [reply game/script-topics.rpy:4413]: Sure, I can adjust that. Enjoy sounds a bit more positive/teasing in this context

**Review by Booplicate — APPROVED** [review]

**Pull request merged**